scripts/setup-local-cluster.sh

```diff
@@ -136,51 +125,52 @@ function cleanup {
 
 trap cleanup SIGINT SIGTERM ERR EXIT
 
-# $1 = rest port
-# $2 = node port
-# $3 = healthcheck port
-# $4 = node data directory
-# $5 = node log file
-# $6 = node id file
-# $7 = host to listen on
-# $8 = OPTIONAL: additional args to hoprd
+# $1 = node id
+# $2 = host to listen on
+# $3 = OPTIONAL: additional args to hoprd
 function setup_node() {
-  local api_port=${1}
-  local node_port=${2}
-  local healthcheck_port=${3}
-  local dir=${4}
-  local log=${5}
-  local id=${6}
-  local host=${7}
-  local additional_args=${8:-""}
+  local node_id=${1?"setup_node requires node id"}
+  local host=${2?"setup_node required host"}
+  local additional_args=${3:-""}
+
+  local api_port p2p_port healthcheck_port dir log id_file safe_args
 
-  log "Run node ${id} on rest port ${api_port} -> ${log}"
+  dir="${tmp_dir}/${node_prefix}_${node_id}"
+  log="${dir}.log"
+  id_file="${dir}.id"
+  api_port=$(( node_api_base_port + node_id ))
+  p2p_port=$(( node_p2p_base_port + node_id ))
+  healthcheck_port=$(( node_healthcheck_base_port + node_id ))
+  safe_args="$(<${dir}.safe.args)"
+
+  api_endpoints+="localhost:${api_port} "
+
+  log "Run node ${node_id} on rest port ${api_port} -> ${log}"
 
   if [[ "${additional_args}" != *"--network "* ]]; then
     additional_args="--network anvil-localhost ${additional_args}"
   fi
 
-  local safe_args=$(<${dir}.safe.args)
   log "safe args ${safe_args}"
   # read safe args and append to additional_args TODO:
   additional_args="${additional_args} ${safe_args}"
 
   log "Additional args: \"${additional_args}\""
 
   env \
-    DEBUG="hopr*" \
+    DEBUG="libp2p*,hopr*" \
```

Comment:
Can we please make this optional or remove it? It floods the logs quite heavily.